### ISSUE TITLE
Does not work with SwiftFormat 0.43.x

### ISSUE THREAD

**Issue by konomae** [garriguv/danger-ruby-swiftformat#26]
danger-swiftformat does not work because the output was changed in SwiftFormat 0.43.😭

This change no longer matches the regular expression.
https://github.com/nicklockwood/SwiftFormat/commit/e723d009203647dcc80592fdcca238203b5e0f72#diff-12d7cfe89917ea40fa2991c6606fb7a7L779
https://github.com/garriguv/danger-ruby-swiftformat/blob/3f9931f2c918dc054b2dda209c13623be4d83dcb/lib/swiftformat/swiftformat.rb#L44

By the way, RUNTIME_REGEX also no longer matches (SwiftFormat != swiftformat).
https://github.com/garriguv/danger-ruby-swiftformat/blob/3f9931f2c918dc054b2dda209c13623be4d83dcb/lib/swiftformat/swiftformat.rb#L59

SwiftFormat's Output

```
swiftformat --dryrun --verbose . 
Running SwiftFormat...
(dryrun mode - no files will be changed.)
Formatting /path/to/swiftformat-test/main.swift
-- rules applied: indent, spaceInsideParens

SwiftFormat completed in 0.16s.
1/1 files would have been formatted.
```

Maybe it's better to use the lint option.🤔

```
swiftformat --lint .
Running SwiftFormat...
(lint mode - no files will be changed.)
/path/to/swiftformat-test/main.swift:3:1: warning: (spaceInsideParens) Remove space inside parentheses.
/path/to/swiftformat-test/main.swift:6:1: warning: (indent) Indent code in accordance with the scope level.
```

**Comment by jazz-mobility** [comment]
+1

**Comment by garriguv** [comment]
@alephao submitted a patch that'll work with SwiftFormat versions > v0.43.

I have released `v0.7.0` of this gem that includes the patch.
